server/routes/search/validationChecks.ts

```diff
@@ -0,0 +1,15 @@
+export const isValidPrisonerNumber = (prisonerNo: string): boolean => {
+  const prisonerNoRegExp = /^[A-Z][0-9]{4}[A-Z]{2}$/
+  const matches = prisonerNo.match(prisonerNoRegExp)
+  return matches !== null
+}
+
+export const extractPrisonerNumber = (search: string): string | false => {
+  const searchTerms = search.toUpperCase().split(' ')
+  return searchTerms.find(term => isValidPrisonerNumber(term)) || false
+}
+
+export const isValidVisitReference = (reference: string): boolean => {
+  const matches = reference.match(/^[a-z]{2}-[a-z]{2}-[a-z]{2}-[a-z]{2}$/)
```

Comment:
Are we using this function?

Reply:
Yes it can be removed, it is not used.
isValidVisitReference removed